### PR TITLE
feat: nft badge new styles

### PR DESCRIPTION
**Pull request by lelenei**
closes #

### CODE REVIEW

File: packages/maskbook/src/components/InjectedComponents/NFT/NFTBadge.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import { NFTAvatarAmountIcon } from '@masknet/icons'
+import { UnionIcon } from '@masknet/icons'
 import { useStylesExtends } from '@masknet/shared'
 import { makeStyles } from '@masknet/theme'
 import { resolveOpenSeaLink } from '@masknet/web3-shared'
 import { CircularProgress, Link, Typography } from '@material-ui/core'
+import BigNumber from 'bignumber.js'
+import classnames from 'classnames'
```
Comment (zhouhanseng) [inline packages/maskbook/src/components/InjectedComponents/NFT/NFTBadge.tsx:7]:
```suggestion
import classNames from 'classnames'
```
Camel case naming

File: packages/maskbook/src/components/InjectedComponents/NFT/NFTBadge.tsx [hunk 2]
```diff
@@ -11,53 +13,72 @@ const useStyles = makeStyles()({
...
     size?: number
 }
+
+function formatPrice(amount: string) {
+    const _amount = new BigNumber(amount ?? '0')
+    if (_amount.isZero()) return '0'
+    if (_amount.isLessThan(1)) return _amount.toFixed(1, 2)
+    if (_amount.isGreaterThan(1e6)) return `${_amount.div(1e6).toFixed(1)}M`
+    if (_amount.isGreaterThan(1e3)) return `${_amount.div(1e3).toFixed(1)}K`
+    return _amount.toFixed(2)
```
Comment (zhouhanseng) [inline packages/maskbook/src/components/InjectedComponents/NFT/NFTBadge.tsx:70]:
```suggestion
    if (_amount.isLessThan(1)) return _amount.toFixed(1, 2)
    if (_amount.isLessThan(1e3)) return _amount.toFixed(2)
    if (_amount.isLessThan(1e6)) return `${_amount.div(1e3).toFixed(1)}K`
    return `${_amount.div(1e6).toFixed(1)}M`
```

makes code more readable

**Comment by guanbinrui** [comment]
Need to be verified by @DimensionDev/qa.

**Review by guanbinrui — APPROVED** [review]

**Comment by lelenei** [comment]
取到symbol, name, price:
image

**Comment by lelenei** [comment]
只取到了price: 
image

**Comment by lelenei** [comment]
取到了name和symbol，没有price:
image

**Comment by lelenei** [comment]
其他情况：
image

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**